models/players.js

```diff
@@ -222,6 +222,17 @@ function mongoInsertPlayers(team_id, team_document){
   });
 }
 
+function mongoBulkInsertPlayers(document){
+    db.open(function(err, db){
+      db.collection("players").insert(document, function(err){
```

Comment:
Why is this an insert not an upsert? Do we really only want to do this once, or set it up to update info each night/week etc.?